plugins/window_size/windows/window_size_plugin.cpp

```diff
@@ -13,17 +13,17 @@
 // limitations under the License.
 #include "window_size_plugin.h"
 
-#include <ShellScalingApi.h>
-#include <windows.h>
-
 #include <VersionHelpers.h>
 #include <flutter/flutter_view.h>
 #include <flutter/method_channel.h>
 #include <flutter/plugin_registrar_windows.h>
 #include <flutter/standard_method_codec.h>
+#include <flutter_windows.h>
+#include <windows.h>
```

Comment:
IIRC having windows.h after VersionHelpers.h broke compilation, which is why they are a separate group. I think maybe putting a comment on the group will prevent the autoformatter from reordering it.